### PR TITLE
Activating Open Collective

### PR DESCRIPTION
**Pull request by monkeywithacupcake**
Hi, I'm making updates for Open Collective. Either you or another core contributor signed this repository up for Open Collective. This pull request adds financial contributors from your Open Collective https://opencollective.com/dts-bundle-generator ❤️

  What it does:
  - adds a badge to show the latest number of financial contributors
  - adds a banner displaying contributors to the project on GitHub
  - adds a banner displaying all individuals contributing financially on Open Collective
  - adds a section displaying all organizations contributing financially on Open Collective, with their logo and a link to their website

P.S: As with any pull request, feel free to comment or suggest changes.

  Thank you for your great contribution to the Open Source community. You are awesome! 🙌
  And welcome to the Open Collective community! 😊

  Come chat with us in the #opensource channel on https://slack.opencollective.com - great place to ask questions and share best practices with other Open Source sustainers!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -192,3 +192,33 @@ but:
 [npm-version-img]: https://badge.fury.io/js/dts-bundle-generator.svg
 [npm-downloads-img]: https://img.shields.io/npm/dm/dts-bundle-generator.svg
 [npm-link]: https://www.npmjs.com/package/dts-bundle-generator
+
+## Contributors
+
+### Code Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
```
Comment (timocov) [inline README.md:200]:
```suggestion
This project exists thanks to all the people who contribute.
```

I don't have contributing file in the repo :)

**Comment by timocov** [comment]
Hey @monkeywithacupcake! Thanks for reaching out with that. Did I understand correctly that these lists will be automatically updates so no one needs to make any changes to get badges/avatars updated, right?

**Comment by monkeywithacupcake** [comment]
They should update automatically. If things look off, please reach out to support on Open Collective Slack.

**Comment by timocov** [comment]
Yeah, thank you!

**Pull request merged**